eng/targets/CSharp.Common.targets

```diff
@@ -52,7 +52,7 @@
     <ImplicitUsings>enable</ImplicitUsings>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(ImplicitUsings)' == 'enable'">
+  <ItemGroup Condition="'$(ImplicitUsings)' == 'enable' or '$(ImplicitUsings)' == 'true'">
```

Comment:
We consistently use `<ImplicitUsings>enable</ImplicitUsings>` and I'm therefore not sure what value this change has. If someone updated a sample project to use `<ImplicitUsings>true</ImplicitUsings>`, we should reject it as unnecessarily inconsistent w/ the rest of the repo.

See [line 52](https://github.com/dotnet/aspnetcore/pull/44665/files#diff-27975d5a8c70351ef594d13df76b6ba6a40c632f5fb38291a5ca6e0caee8957aR52) just above for example.

Reply:
Other option: we could say with that change we don't have to reject anything. And nobody will have to search for the reason why the documented value doesn't work.

Reply:
The code here doesn't change whether an `$(ImplicitUsings)` setting works or not. I simply controls a few `using`s that are added. In any case, I'd rather **not** introduce the `enable` versus `true` inconsistency in the future. We've chosen one and it's easily understood.

Reply:
The issue is with transitive consumers. For example, in a conversation with @ReubenBond - we were discussing that if a consuming project other than the internal projects in your control, enable implicit usings with `<ImplicitUsings>true</ImplicitUsings>` this wouldn't work.

Reply:
That can't happen; this file is used only when building projects in this repository

Reply:
That's what I assumed too, @ReubenBond made it seem like that's not the case.

Reply:
> That's what I assumed too, @ReubenBond made it seem like that's not the case.

Because in our case it isn't: we are shipping some using directives this way

Reply:
&hellip; but we never ship this file; it's not included in any shipping package.

Reply:
Setting `$(ImplicitUsings)` in our templates would only interact w/ this code if we generated projects and allowed this code to be imported. Project template tests (for example) avoid that by generating projects under `$env:Temp`.